packages/admin-ui/ui/src/utils/tree-helpers.ts

```diff
@@ -0,0 +1,13 @@
+export const flattenCategoryTree = (rootCategories) => {
```

Comment:
suggestion: move to `packages/admin-ui/ui/src/domain/categories/utils/` since it's related to that domain